project/AAI-RoadMap.md

```diff
@@ -0,0 +1,55 @@
+# Animal-AI RoadMap
```

Comment:
I think having the website stay out of this would help in maintenance and effort costs. As i pointed in the bottom of the file, it is a WIP so not complete. I will be having a meeting with Lucy to define our roadmap and vision in the coming days :)